plexpy/graphs.py

```diff
@@ -909,3 +913,114 @@ def get_stream_type_by_top_10_users(self, time_range='30', y_axis='plays'):
                   'series': [series_1_output, series_2_output, series_3_output]}
 
         return output
+
+    @staticmethod
+    def data_by(time_range='30', t='month', refresh=False, **kwargs):
+
+        """ Filter all files from plex
+
+            Args:
+                time_range(string, optional): ''
+                t(string): 'raw, day, date, month, hour'
+
+            Returns:
+                dict:
+                    ```
+                    {'date': [t], 'movies': [ints], 'track': [ints], 'episode': [ints]}
+                    ```
+
+
+        """
+        # Placeholders
+        all_files = None
+        limit = None
+        movie_list = []
+        episode_list = []
+        track_list = []
+        match_list = []
+
+        json_file = os.path.join(plexpy.CONFIG.CACHE_DIR, 'media_info_graphs.json')
+
+        try:
+
+            with open(json_file, 'r') as f:
+                all_files = json.load(f)
+        except Exception as e:
+            logger.debug('Failed to read %s %s' % (json_file, e))
+            return []
+
+        # Used for debugging
+        if t == 'raw':
+            return all_files
+
+        base = datetime.datetime.now().date()
+
+        # Make list with the correct matches
+        if t == 'hour':
+            match_list = [d.hour for d in helpers.get_time_range('hours', n=-24, raw=True)]
+
+        elif t == 'day':
+            # dayofweek
+            match_list = [d.weekday() for d in helpers.get_time_range('days', n=-7, raw=True)]
```

Comment:
I figured out why. @OmgImAlexis was right. It is because of the JS `Date()` function and the timezone conversions using the `YYYY-MM-DD` date format, whereas all the other graphs use the UNIX timestamp and don't have the conversion problem.

I suggest you do all the dates with timestamps so we don't have weird issues with timezones.